CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Crash report email subject gives host name
 - Export general case report to PDF using PDFKit instead of WeasyPrint
 - Do not include coverage report in PDF case report since they might have different orientation
+- Reduce code complexity (parse/ensembl.py)
```

Comment:
Nice! Don't forget to check the relative checkboxes in #2984, so we know which functions have been rewritten!

Reply:
Will do!